AK/Statistics.h

```diff
@@ -29,6 +29,28 @@ requires(IsArithmetic<T>) class Statistics {
     T const sum() const { return m_sum; }
     float average() const { return (float)sum() / size(); }
 
+    T const min()
+    {
+        T min_v = m_values[0];
```

Comment:
how about `minimum` as the variable name? `minimum_value` seems too verbose and `min_v` seems a little too shortened.
```suggestion
        T minimum = m_values[0];
```